packages/core/src/common/keys.ts

```diff
@@ -148,7 +172,7 @@ const MODIFIERS: Key[] = [];
 export namespace Key {
 
     export function isKey(arg: any): arg is Key {
-        return (<Key>arg).code !== undefined && (<Key>arg).keyCode !== undefined;
+        return arg && (<Key>arg).code !== undefined && (<Key>arg).keyCode !== undefined;
```

Comment:
I just would like to understand this: do not we have an explicit `undefined`/`null` check enabled?

Reply:
just of the topic: such code looks dangerous, it can cause side effects by calling code or keyCode, better:
```ts
return !!arg && ('code' in arg) && ('keyCode' in arg);
```

Reply:
Why is this dangerous? That is the recommended way according to the [TS documentation](https://www.typescriptlang.org/docs/handbook/advanced-types.html), did I miss something? What could be the problem, could you please give us an example, @akosyakov?

Reply:
- `('code' in arg)` checks only that arg has a property with `code` name
- `(<Key>arg).code !== undefined` calls `code` property and checks whether the result is of `undefined` type - it can be also bogus if there is code property which returns `undefined` value